docs/how-to-guides/dependabot-auto-merge.md

```diff
@@ -0,0 +1,117 @@
+# Dependabot with auto-merge
+
+[working-with-dependabot]: https://docs.github.com/en/code-security/dependabot/working-with-dependabot
+[automating-dependabot]: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions
+[configure-dependabot-yaml]: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+[github-cli]: https://cli.github.com/
+
+[Dependabot][working-with-dependabot] is a security tool offered by GitHub.
+Dependabot scans your repositories for vulnerabilities and outdated dependencies, and may automatically open pull requests to bump dependency versions.
+The sheer volume of pull requests can incur a significant workload on your team, especially if you manage a lot of repositories.
+
+By completing this guide, Dependabot will automatically fix your insecure or outdated dependencies, and the changes will automatically get merged into your main branch.
+
+## Prerequisites
+
+* [GitHub command-line interface][github-cli] installed.
+
+## Enable Dependabot
+
+The contents of this file will depend on your project requirements. Do not use this file as-is.
+Please see [dependabot.yaml configuration syntax][configure-dependabot-yaml] for detailed instructions on how to configure Dependabot.
+
+!!! note ".github/dependabot.yaml"
+
+	```yaml
+	version: 2
+	updates:
+	  - die: &I didn't edit my config file
```

Comment:
Hva betyr det her? Er det bare for å hindre at de kopierer filen? For da tenker jeg det er like greit å ikke ha et eksempel der.

Reply:
De må jo ha den filen uansett, synes det er like greit å vise sånn cirka hvordan den skal se ut fremfor å ikke ha noe doc som helst. Åpen for andre forslag.